docker/x86_64-musl/Dockerfile

```diff
@@ -1,12 +1,3 @@
-FROM debian:stretch
+FROM alpine:edge
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends nettle-dev musl-tools \
-        make file wget netcat-traditional sqlite3 git ca-certificates ssh
-
-# Install ghr for GitHub Releases: https://github.com/tcnksm/ghr
-RUN wget https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_amd64.tar.gz && \
```

Comment:
Do we want to keep this in the image?

Reply:
Are we actually using it?

Reply:
We are not currently, but at some point we planned on using it. It's in all of the other images, so might as well keep it for consistency. We can later determine if we want to remove it from all of the images.